src/main/java/com/endava/soa_5ed/task1_basics/Ex1.java

```diff
@@ -0,0 +1,17 @@
+package com.endava.soa_5ed.task1_basics;
+
+public class Ex1 {
```

Comment:
You could change the class name to something like: SecondsConverter
It might be easier to maintain the code

Reply:
Done.